src/loaders/loadSound.ts

```diff
@@ -0,0 +1,68 @@
+import { utils, extensions, ExtensionType } from '@pixi/core';
+import { LoadAsset, LoaderParserPriority } from '@pixi/assets';
+import { supported } from '../utils/supported';
+import { Options, Sound } from '../Sound';
+
+import type { LoaderParser } from '@pixi/assets';
+import { getInstance } from '../instance';
+
+/** Get the alias for the sound */
+const getAlias = (asset: LoadAsset) =>
+{
+    const url = asset.src;
+
+    return (asset as any)?.alias?.[0] ?? utils.path.basename(url, utils.path.extname(url));
```

Comment:
@Zyie do you know what `alias` and `format` are not typed in LoadAsset, but seem to be available.

Reply:
Looks like `alias` and `format` gets added by the `Resolver` class [here](https://github.com/pixijs/pixijs/blob/41087b16732769b0f6170114a2fcd75a4dd61bde/packages/assets/src/resolver/Resolver.ts#L333)

Seems like the typings need to be improved here

Reply:
Also these only get added if there is an entry in the resolver

So this example would have `alias`
```ts
Assets.add('sound', 'path/to/sound.mp3')
Assets.load('sound')
```

But this would not
```ts
Assets.load('path/to/sound.mp3')
```